src/nl/hannahsten/texifyidea/completion/handlers/LatexMathInsertHandler.kt

```diff
@@ -11,7 +11,7 @@ import nl.hannahsten.texifyidea.lang.commands.Argument
 class LatexMathInsertHandler(val arguments: List<Argument>?) : InsertHandler<LookupElement> {
```

Comment:
Class seems unused? As the functionality is all covered by LatexCommandInsertHandler